package.json

```diff
@@ -20,32 +20,26 @@
   "dependencies": {
     "@brightspace-ui/core": "^2",
     "@polymer/polymer": "^3.0.0",
+    "d2l-fetch": "Brightspace/d2l-fetch.git#semver:^2",
```

Comment:
This shouldn't have been a dev-dependency, it's imported by `SirenAction.js` (unknown if it actually needs to be).